radboudumc-template/templates/new/publications.html

```diff
@@ -0,0 +1,123 @@
+{% extends "base.html" %}
+{% block title %}{{ page.title }}{%endblock%}
+{% block description %}{{ SITENAME }} publications overview.{% endblock %}
+{% block page_picture %}{{ SITEURL }}/{{SITE_PICTURE}}{% endblock %}
+{% block content %}
+    {% include 'blocks/breadcrumbs.html' %}
+    <div class="container">
+    <h1>{{ page.title }}</h1>
+
+
+
+    {% set list_years = [] %}
+    {% set list_pub_type = [] %}
+
+
+    {% if bibkeys_html %}
+    {% for bibkey in bibkeys_html.keys() %}
+        {# Getting list of years #}
+        {% if bibkeys_html[bibkey]['year'] not in list_years %}
+            {% if list_years.append(bibkeys_html[bibkey]['year']) %}{% endif %}
+        {% endif %}
+        {# Getting list of pub_type #}
+        {% if bibkeys_html[bibkey]['pub_type'] not in list_pub_type %}
+            {% if list_pub_type.append(bibkeys_html[bibkey]['pub_type']) %}{% endif %}
+        {% endif %}
+    {% endfor %}
+    {% endif %}
+    <style>
+      input[name="year-selection"]:checked + label {
+      font-weight: bold;
+      }
+      input[name="year-selection"]:hover + label {
+      font-weight: bold;
+      text-decoration: none;
+      }
```

Comment:
Please move this to the css file. Also, the buttons are now missing a `cursor: pointer` tag and jump when hovered.

Reply:
done, jumpy not fixed yet